app/main.py

```diff
@@ -1,3 +1,54 @@
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: float | "Distance") -> "Distance":
```

Comment:
The type hint for other here and in __iadd__ should be int | float | Distance to support integer operations like distance1 + 10, as shown in the requirements